scripts/integration-test

```diff
@@ -85,8 +90,17 @@ trap "kill $pid_controller $pid_replica1 $pid_replica2 \
        $pid_backing_replica1 $pid_backing_replica2 \
        $pid_upgrade_replica1 $pid_upgrade_replica2" EXIT
 
+# start minio server to test s3 backup and set credential keys
+export MINIO_ACCESS_KEY=test-access-key
+export MINIO_SECRET_KEY=test-secret-key
+# create default bucket
+mkdir -p /data/backupbucket
+# run minio server
+nohup /usr/bin/minio server /data &
```

Comment:
Why need to use `nohup` here?

Reply:
Using nohup to make minio server run in background, if not it would to start a server in frontend and block current bash.

Reply:
You already put the running process in the background using `&`